frontend/src/app/tasks/components/TaskBox.tsx

```diff
@@ -68,18 +74,39 @@ export const TaskBox = ({ task, projects, taskTypes, tasks }: TaskProps) => {
             <Typography textColor="#1E2AA5" fontWeight="bold">
               {task.startTime}-{task.endTime} ({timeDifference()})
             </Typography>
+            {expandedTask && (
+              <>
+                <Typography textColor="#1E2AA5">{task.description}</Typography>
+                <Typography textColor="#1E2AA5">{task.story}</Typography>
+              </>
+            )}
           </>
         )}
       </Box>
       <Box display="flex" gap="8px">
-        <IconButton onClick={() => setEditMode(true)}>
+        <IconButton
+          onClick={() => {
+            setEditMode(true)
+            setExpandedTask(false)
+          }}
+        >
           <Edit24Filled color="#2f3338" />
```

Comment:
At some point, I think we should extract all hard-coded hex values and use a stylesheet and variables (or some other centralized method) for colors/styles. If we were to ever want to change our icon colors, for instance, it would be easier to change the value once than across multiple files. I won't hold the PR on this, but it is something we probably want to consider doing soon before we add too much more code. The theme (colors, fonts, border radius, etc.) that we are using in the app will be part of what we want to use with other applications, so we will want to implement it in a way that can be shared.